modules/govuk/manifests/apps/email_alert_api.pp

```diff
@@ -132,8 +140,10 @@
   $email_alert_auth_token = undef,
   $email_service_provider = 'NOTIFY',
   $email_address_override = undef,
-  $email_address_override_whitelist = undef,
-  $email_address_override_whitelist_only = false,
+  $email_address_override_allowlist = undef,
+  $email_address_override_allowlist_only = false,
+  $email_address_override_whitelist = $email_address_override_allowlist,
+  $email_address_override_whitelist_only = $email_address_override_allowlist_only,
```

Comment:
I'd suggest you can delete the parameters for the whitelist options now (and the associated hiera config)